index.html

```diff
@@ -187,15 +187,15 @@ <h4 class="text-proxima text-20 mb-3">
               <p>This is a greate resource to get you started on project managment and stuff. This is a greate resource to get you started on project managment and stuff.</p>
             </div>
           </div>
-    
           <a href="" class="btn btn-lg btn-outline-danger mx-auto read-more-btn btn-block">
             <span class="mr-1">Get More Resources</span>
-            <svg xmlns="http://www.w3.org/2000/svg" width="19" height="12" viewBox="0 0 19 12">
+            <svg xmlns="http://www.w3.org/2000/svg" width="19" height="12" viewbox="0 0 19 12">
```

Comment:
Why did you have to do this change?

Reply:
It was marked in VS Code as an error for some reason